tests/timeseries/test_sort_timestamps_monotonically.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import pandas as pd
+from random import randint
+from janitor.timeseries import sort_timestamps_monotonically
+
+
+# Random data for testing
+@pytest.fixture
+def timeseries_dataframe() -> pd.DataFrame:
+    """
+    Returns a time series dataframe
+    """
+    ts_index = pd.date_range("1/1/2019", periods=1000, freq="1H")
+    v1 = [randint(1, 2000) for i in range(1000)]
+    test_df = pd.DataFrame({"v1": v1}, index=ts_index)
+    return test_df
+
+
+@pytest.mark.timeseries
+def test_sort_timestamps_monotonically(timeseries_dataframe):
+    """Test that sort_timestamps_monotonically works as expected."""
+    from sklearn.utils import shuffle
+
+    # Increasing direction
+    df = shuffle(timeseries_dataframe)
+    df1 = sort_timestamps_monotonically(df)
+    assert df1.equals(timeseries_dataframe)
```

Comment:
I forget if the `assert` pattern is used in the other PR you worked on, but I think across the `pyjanitor` codebase, we have used `pd.testing.assert_frame_equal(df1, df2)` instead of `assert df1.equals(df2)`. Can you check to see what the pattern is used elsewhere, and follow that for consistency please?